src/pages/PageNotFound.jsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
+
+function PageNotFound() {
+  return (
+    <div className="flex   text-white flex-col items-center justify-center  p-10">
+      <img
+        className="w-96 mx-auto "
+        src="src\assets\img\page_not_found_img\404-error.png"
```

Comment:
Change it to `src="src/assets/img/page_not_found_img/404-error.png"` to fix this error